File: AutoIt.Foundation/AutoIt.Foundation.Web.Scripts/CodeEdit/LangAnaly/Model/CharSet.ts
///<reference path="EgtEntityBase.ts"/>>
namespace CodeEdit.LangAnaly.Model {
   //×Ö·û¼¯(DFA±ß)
   export  class  CharSet extends  CodeEdit.LangAnaly.Model.EgtEntityBase{
       //×Ö·û¼¯Ïî
       public Group: List<CodeEdit.LangAnaly.Model.CharSetItem> = new List<CodeEdit.LangAnaly.Model.CharSetItem>(); 
    }
}

File: AutoIt.Foundation/AutoIt.Foundation.Common/LangAnaly/Model/DFA/Egt/CharSet.cs
using System.Collections.Generic;

namespace Compiler.Egt
{
    public class CharSet:EgtEntityBase
    {
        public List<CharSetItem> Group { get; set; }

        public class CharSetItem
        {
            public int Start { get; set; }
            public int End { get; set; }
        }
    }
}